test/objects.js

```diff
@@ -82,7 +82,10 @@
     assert.deepEqual(_.pairs({one: 1, two: 2}), [['one', 1], ['two', 2]], 'can convert an object into pairs');
     assert.deepEqual(_.pairs({one: 1, two: 2, length: 3}), [['one', 1], ['two', 2], ['length', 3]], '... even when one of them is "length"');
     if (typeof Map === 'function') {
-      assert.deepEqual(_.pairs(new Map([['one', 1], ['two', 2]])), [['one', 1], ['two', 2]], '... or when object is a Map');
```

Comment:
I read in a previous PR that IE11 does not support initializing Maps with arrays in the constructor, hence the refactor here.